spec/features/idv/inherited_proofing/inherited_proofing_cancel_spec.rb

```diff
@@ -0,0 +1,257 @@
+require 'rails_helper'
+
+# Simulates a user (in this case, a VA inherited proofing-authorized user)
+# coming over to login.gov from a service provider, and hitting the
+# OpenidConnect::AuthorizationController#index action.
+def send_user_from_service_provider_to_login_gov_openid_connect(user)
+  expect(user).to_not be_nil
+  # NOTE: VA user.
+  visit_idp_from_oidc_va_with_ial2
+end
+
+def complete_idv_steps_up_to_inherited_proofing_get_started_step(user, expect_accessible: false)
+  unless current_path.match?(/inherited_proofing[?|\/].*get_started/)
+    complete_idv_steps_before_phone_step(user)
+    click_link t('links.cancel')
+    click_button t('idv.cancel.actions.start_over')
+    expect(page).to have_current_path("#{idv_inherited_proofing_path}/get_started")
+  end
+  expect(page).to be_axe_clean.according_to :section508, :"best-practice" if expect_accessible
+end
```

Comment:
Would these make sense to include in the existing `inherited_proofing_helper.rb` ? I see similar methods there (and similar helpers for in-person and standard identity verification).

Reply:
Normally, I would agree; however, all the other tests don't necessarily require session (`decorated_session` specifically) to be set up properly, so these inline (in-file) helpers are specific to this test suite. IOW, no other IP tests can use this except for this feature. I will move these if this changes.